repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/classloader/ClassLoaderTest.groovy

```diff
@@ -92,7 +82,8 @@ class ClassLoaderTest extends ReposeValveTest {
         repose.start(killOthersBeforeStarting: false,
                 waitOnJmxAfterStarting: false)
 
-        repose.waitForNon500FromUrl(url)
+        reposeLogSearch.awaitByString("Repose ready", 1, 30)
```

Comment:
It's a messy method, but you could use `waitUntilReadyToServiceRequests(checkLogMessage = true)` instead. It has a bit more logic that kicks in if the log message isn't found, but it's a method that gets used elsewhere, so as least we get that consistency and consolidation.

As a side note, we really need to fix up these methods that wait for Repose to be ready.